go/fixchain/fix_error.go

```diff
@@ -14,18 +17,20 @@ const (
 	ParseFailure
 	CannotFetchURL
 	FixFailed
+	PostFailed
 	LogPostFailed
 	VerifyFailed
 )
 
 // FixError is the struct with which errors in the fixing process are reported
 type FixError struct {
-	Type  errorType
+	Type  errorType           // The type of error
 	Cert  *x509.Certificate   // The supplied leaf certificate
 	Chain []*x509.Certificate // The supplied chain
 	URL   string              // URL, if a URL is involved
 	Bad   []byte              // The offending bytes, if applicable
```

Comment:
Meh. Anyone can abuse any struct they want. Probably best to document it is a cert. Especially since we dump it as one...